src/box/lua/connpool.lua

```diff
@@ -118,7 +172,7 @@ function pool_methods.connect_to_multiple(self, instances, opts)
 
     local delay = WATCHER_DELAY
     local timeout = opts.timeout or WATCHER_TIMEOUT
-    local connect_deadline = clock.monotonic() + timeout
+    local connect_deadline = fiber.clock() + timeout
```

Comment:
First, it looks unrelated to the other changes. Next, it is still compared to `clock.monotonic()` below in the loop condition.